app/src/processing/app/Editor.java

```diff
@@ -1374,6 +1374,24 @@ public void actionPerformed(ActionEvent e) {
 
     menu.addSeparator();
 
+    JMenuItem increaseFontSizeItem = newJMenuItem(tr("Increase Font Size"), '=');
```

Comment:
Why didn't you choose `ctrl + '+' ` as increase shortcut?  The `=` version doesn't seem to work with an Italian keyboard (since `=` can only be reached with `Maiusc` modifier)